docs/README.md

```diff
@@ -31,20 +31,26 @@ For documentation on code: [**JSDocs**](https://engineering.cerner.com/carbon-gr
 
 ## Colors
 
+-   According to new accessibility guidelines, Carbon colors must meet contrast ratio of 3:1. Pink color is added additionally and should update all colors to meet 3:1 contrast ratio requirement.
```

Comment:
Is there a link to the accessibility guidelines?

Reply:
Here is the [link](https://cernerprod-my.sharepoint.com/:w:/g/personal/lb047858_cerner_net/EVJgH70BOuhIpp3auhLe6qUBtNoyeiBwinXiAXKUKqYiBg?e=0tY2NF) for accessibility guidelines.